jipp-core/bin/genTypes.py

```diff
@@ -776,7 +780,7 @@ def fix_ktypes(type, syntax, name, group_name = ''):
         intro = 'IntOrIntRangeType%s(' % set
         type['ktype'] = 'IntOrIntRange'
         type['kalt'] = [ 'Int', 'IntRange' ]
-    elif re.search('^integer(\([0-9MINAX: -]*\))?$', syntax):
+    elif re.search('^integer\s?(\([0-9MINAX: -]*\))?$', syntax):
```

Comment:
\s will match whitespace, this is added to handle _Integer(0:MAX)_ and _Integer (0:MAX)_

![image](https://github.com/user-attachments/assets/e66176dc-9aec-4e7c-bbee-15bd027d9447)